repository-data/webfiles/src/main/resources/site/freemarker/hee/pages/homepage-main.ftl

```diff
@@ -1,19 +1,41 @@
+<#-- @ftlvariable name="document" type="uk.nhs.hee.web.beans.HomePage" -->
 <#include "../../include/imports.ftl">
+<#import "../macros/components.ftl" as hee>
 <@hst.defineObjects />
-
 <@hst.setBundle basename="uk.nhs.hee.web.homepage"/>
```

Comment:
@thojava - Could we get rid of `/content/documents/administration/labels/homepage` resource bundle ?

Reply:
Right. We do not use this any more so let me clean it.